src/W3C.xml

```diff
@@ -0,0 +1,58 @@
+<SPDX name="W3C Software Notice and License (2002-12-31)" identifier="W3C" osi-approved="true">
+  <urls>
+    <url>http://www.w3.org/Consortium/Legal/2002/copyright-software-20021231.html</url>
+    <url>http://www.opensource.org/licenses/W3C</url>
+  </urls>
+  <notes>This license was released: 13 December 2002. The standard header is provided via a link from the original
+         license page.</notes>
+  <header>Copyright (C) 
+    <alt name="year" match=".+">[$date-of-software]</alt> World Wide Web Consortium, (Massachusetts Institute of
+    Technology, European Research Consortium for Informatics and Mathematics, Keio University). All Rights Reserved.
+    This work is distributed under the W3C® Software License in the hope that it will be useful, but WITHOUT ANY
+    WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+  </header>
+  <license>
+    <title>
+      <p>W3C SOFTWARE NOTICE AND LICENSE</p>
+    </title>
+    <optional>
+      <p>This work (and included software, documentation such as READMEs, or other related items) is being
+         provided by the copyright holders under the following license.</p>
+      <p>License</p>
+      <p>By obtaining, using and/or copying this work, you (the licensee) agree that you have read, understood,
```

Comment:
I mostly agree -- I think the license actually starts at line 19.

I think the confusion comes from the multiple meanings of the word "license," which can mean the set of rights granted to a licensee within a contract, or colloquially can refer to the whole contract itself. I think in this case we have the former. If you look at the [original page](https://www.w3.org/Consortium/Legal/2002/copyright-software-20021231.html), it looks like there is a license-granting section of the agreement with header "License" and a warranty-disclaiming section of the license with header "Disclaimers."

This is somewhat undercut by the "Notes" section at the end which seems like it is _not_ intended to be part of the agreement, but I think that's a separate question and that words to the effect of "this work is provided by the copyright holders under the following license" is a part of the agreement.